mmv1/products/gkebackup/RestorePlan.yaml

```diff
@@ -0,0 +1,452 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'RestorePlan'
+base_url: 'projects/{{project}}/locations/{{location}}/restorePlans'
+create_url: projects/{{project}}/locations/{{location}}/restorePlans?restorePlanId={{name}}
+update_verb: :PATCH
+update_mask: true
+description: |
+  Represents a Restore Plan instance.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/kubernetes-engine/docs/add-on/backup-for-gke'
+  api: 'https://cloud.google.com/kubernetes-engine/docs/add-on/backup-for-gke/reference/rest/v1/projects.locations.restorePlans'
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  method_name_separator: ':'
+  parent_resource_attribute: 'name'
+  base_url: projects/{{project}}/locations/{{location}}/restorePlans/{{name}}
+  import_format:
+    [
+      'projects/{{project}}/locations/{{location}}/restorePlans/{{name}}',
+      '{{name}}',
+    ]
+timeouts: !ruby/object:Api::Timeouts
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'gkebackup_restoreplan_all_namespaces'
```

Comment:
This first example needs to have `primary_resource_name` for the iam test template to work properly.